test/nondefaulte2e/suite_test.go

```diff
@@ -103,6 +109,7 @@ var _ = BeforeSuite(func() {
 	Expect(configv1.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
 	Expect(templatev1.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
 	Expect(appsv1.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
+	Expect(argoapi.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
```

Comment:
```suggestion
Expect(argov1alpha1api.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
Expect(argov1beta1api.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())	
```

Make this change and import the required deps using 
```go
argov1alpha1api "github.com/argoproj-labs/argocd-operator/api/v1alpha1"
argov1beta1api "github.com/argoproj-labs/argocd-operator/api/v1beta1"
```
in the import section at the start of the file.

Reply:
Applied